NEWS.md

```diff
@@ -1,4 +1,15 @@
-## 2.2.0 - Unreleased
+## 2.3.0 - Unreleased
+
+## 2.2.0 - Released (Quesnelia R1 2024)
+The primary focus of this release was error handling improvements and spring version upgrade
+
+[Full Changelog](https://github.com/folio-org/mod-ebsconet/compare/v2.1.0...v2.2.0)
+
+### Stories
+* [MODEBSNET-70] (https://folio-org.atlassian.net/browse/MODEBSNET-70) - Add error code if default note type "General note" is not exists
```

Comment:
add this story probably missied https://folio-org.atlassian.net/browse/MODEBSNET-73, otherwise would be confusing mentioned two times spring-boot version changed but there is no story about it

Reply:
Please check dependencies section. We do not mention stories updating only pom.xml

Reply:
added after discussing 1-1

Reply:
Nice:)